test/mocha/glob.js

```diff
@@ -21,4 +21,14 @@ describe("minify() with input file globs", function() {
         });
         assert.strictEqual(result.code, 'var print=console.log.bind(console);print("qux",function(n){return 3*n}(3),function(n){return n/2}(12)),function(n){print("Foo:",2*n)}(11);');
     });
+    it("should throw with non-matching glob string", function() {
+        assert.throws(function() {
+            Uglify.minify("test/input/issue-1242/blah.*");
+        }, "should throw file not found");
```

Comment:
This original behavior is patterned after the UNIX shell. Globs that don't match in UNIX shells return the original glob, rather than raising an exception. I'd like to retain that behavior.